### PR TITLE
Removes three questions

### PR DESCRIPTION
**Pull request by jfmercer**
* Removes the environment, class instance, and old app name questions
  and their corresponding template variables
* Closes Issue #55

### CODE REVIEW

File: lib/templates/immutable.yaml.mustache [hunk 1]
```diff
@@ -12,6 +12,7 @@
          instance_type="{{=<% %>=}}{{ instance_type }}<%={{ }}=%>"
          keypair='{{{ keypair }}}'
          instance_tags='{"Environment":"{{{ environment }}}","Class":"{{{ className }}}-immutable","Name":"{{{ appName }}} (immutable)"}'
+         instance_tags='{"Environment":"environment","Class":"class-immutable","Name":"{{{ appName }}} (immutable)"}'
```
Comment (gavD) [inline lib/templates/immutable.yaml.mustache]: looks like a duplicate line to the above?

File: lib/templates/provision.sh.mustache [hunk 2]
```diff
@@ -11,10 +11,10 @@ ssh-add -l
 . ./inventory/aws_keys
 
 # Tag any existing {{ app }} instances as being old
-ansible-playbook tag-old-nodes.yaml --limit tag_Environment_{{{ environment }}}
+ansible-playbook tag-old-nodes.yaml --limit tag_Environment_old
```
Comment (gavD) [inline lib/templates/provision.sh.mustache]: I'm still not sure what we should call this tag... Giving the user control over it is a bit confusing, but `tag_Environment_old` is not the answer either...

File: lib/templates/immutable.yaml.mustache (hunk 1 above)
Comment (gavD) [inline lib/templates/immutable.yaml.mustache]: we can lose class and environment entirely. Something like:

`instance_tags='{"Name":"{{{ appName }}} (immutable)"}'`

Comment (jfmercer) [inline lib/templates/immutable.yaml.mustache]: That's from a bad rebase. Nice catch.

**Pull request closed without merging**